Server documentation #63

smistad opened the pull request:
Added server documentation and performance optimization information to the README

etiennegaucher reviewed (approved):
Very interesting! Should we update the requirements.txt file? It seems to be out of date.

smistad commented:
> Very interesting! Should we update the requirements.txt file? It seems to be out of date.

I have updated the requirements.txt file now as well, and added some missing optional packages for server configuration.
